dashboard/templates/commands.ejs

```diff
@@ -16,7 +16,7 @@ bot.commands.forEach((command) => {
       <% for (const command of help[category]) { %>
         <div class="card">
           <h3 class="card-header"><%= command.help.name %></h3>
-          <div class="card-body" style="height: 100px;">

```

Comment:
Revert this change, it makes the command page look unorganized and cluttered.
[Current](https://i.imgur.com/vAYObto.png)
[Your proposed changes](https://i.imgur.com/qKYbI3q.png)

Reply:
If description is long then it overlaps the footer, so removed the height

Reply:
Whilst true, it gave a uniformed appearance.

Maybe look into limiting characters/words so they fit without causing issues.

Reply:
max 60 characters allowed